roles/virtualbmc/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
   ansible.builtin.file:
     path: "{{ item.key }}"
     state: directory
-    mode: "{{ item.mode | default(omit) }}"
+    mode: "{{ item.mode | default("0755") }}"
```

Comment:
```suggestion
    mode: "{{ item.mode | default('0755') }}"
```

Need to use single quotes inside double quotes

Reply:
That should fix the pre-commit failure

Reply:
and the virtualbmc failure as well :)

Reply:
Thanks 🤦🏽‍♀️  I missed this